src/main/java/com/flipkart/yak/util/PromptCompactionUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.flipkart.yak.util;
+
+import com.flipkart.yak.commons.ScheduleUtils;
+import com.flipkart.yak.config.CompactionContext;
+import com.flipkart.yak.config.CompactionSchedule;
+import com.flipkart.yak.config.PromptCompactionRequest;
+import com.flipkart.yak.config.PromptSchedule;
+
+public class PromptCompactionUtil {
+
+    public static CompactionContext getCompactionContext(PromptCompactionRequest promptCompactionRequest) {
```

Comment:
Can it be part of any other Utility class. 
Util class is just a toolBox, you don't need granular specialisation for Util classes.

Reply:
moved to commons